src/components/stack-chart/Canvas.js

```diff
@@ -345,6 +348,13 @@ class StackChartCanvasImpl extends React.PureComponent<Props> {
             intW + BORDER_OPACITY,
             intH
           );
+
+          if (searchStringsRegExp && searchStringsRegExp.test(text)) {
+            ctx.strokeStyle = 'green';
```

Comment:
I was thinking that the color could be somewhat softer. We can try the mozilla blue `BLUE_60` for example. You can import it like this:
```js
import { BLUE_60 } from 'photon-colors'
```
and use it directly.